packages/trader/src/App/Components/Form/TimePicker/time-picker.tsx

```diff
@@ -34,22 +46,22 @@ const TimePicker = observer(
             setIsOpen(!is_open);
         };
 
-        const handleChange = arg => {
+        const handleChange = (arg: string | React.ChangeEvent<HTMLInputElement>) => {
             // To handle nativepicker;
-            const value = typeof arg === 'object' ? arg.target.selected_time : arg;
```

Comment:
selected_time does not exist in arg.target when it's an object, value is the correct value.